### ISSUE TITLE
Auto-detect AxiDraw model

### ISSUE THREAD

**Issue by haschdl** [evil-mad/axidraw#122]
*Background* 
Currently we have to manually choose the Axidraw model in Inkscape, done via Extensions -> AxiDraw Control -> Options -> Config. The default value is *AxiDraw V2 or AxiDraw V3* 
I own an AxiDraw SE/A3 therefore I change the setting to *AxiDraw V3/A3 or SE/A3*. For reasons unknown to me, Inkscape is not persisting the choice.

*Consequence*
I have in multiple occasions lost time, paper and ink by AxiDraw Control resetting to V2, and the plot is cropped. AxiDraw then shows a message that it hit a limit (sorry I couldn't copy the exact message). This is *very* frustrating, tiresome and wasteful.

*Suggestion(s)*
There are multiple ways to have a better experience, but I believe having to manually choose the model should be removed  altogether.  
- The AxiDraw Control software should auto-detect the model and/or  
- Show a warning that the drawing is off-limits BEFORE plot starts, with option to cancel it

My setup:
* MacOS
* AxiDraw software version 3.1
* Inkscape version 1.1.2

**Comment by oskay** [comment]
Inkscape saves settings when you use **File>Quit**. If you have important changes like these that you want it to remember, you might consider making those changes then quitting and re-opening to verify that they are saved properly.

I would recommend that you run plots in preview mode first, to make sure that everything looks good before starting a plot; you may find that other settings such as optimizations or page rotation are not how you want them either, since you are having trouble with settings. 

If you would like to default to SE/A3 within Inkscape-- permanently removing the V2 option -- you can edit the axidraw.inx file that sets them:

A. Open Inkscape
B. From the menu, select **Inkscape > Preferences**
C. Select System from the list on the left panel of the preferences window.
D. On the right-hand panel is a list of folder locations. Click the "Open" button to the right of the "User extensions" folder location. This will open the extensions folder on your computer.
E. In that folder, locate axidraw.inx and open it in a plain text editor such as TextEdit
F. Locate the line that says `<_option value="1">AxiDraw V2 or AxiDraw V3</_option>`, and delete it
G. Save the file
H. Restart Inkscape

(If you would prefer to leave that option, you can move it after the "AxiDraw V3/A3 or SE/A3" line; the first option listed will be the default.)

There is not presently any software method that can auto-detect the AxiDraw model; it is possible that we could add one at some point, but it would require a firmware update and/or manually setting the model (on existing units) in order to work. It would also require that the model would be re-set whenever the firmware is updated, which is a significant liability.

**Comment by haschdl** [comment]
thank you @oskay , given the alternatives and the impact of making changes to the firmware, I agree this feature request is low priority and I propose to close the issue.

**Comment by oskay** [comment]
Thank you for your understanding. We will keep it in mind. It's a *very* good idea if we can find a way to implement it well.
